packages/api-demo/src/examples/LCDCosmos.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+
+import { cosmos } from '@regen-network/api';
+import { QueryAllBalancesResponseSDKType } from '@regen-network/api/types/codegen/cosmos/bank/v1beta1/query';
+import { PageRequest } from '@regen-network/api/types/codegen/helpers';
+
+export function LCDCosmos(): React.ReactElement {
+  const [result, setResult] = useState<
+    QueryAllBalancesResponseSDKType | undefined
+  >(undefined);
+  const [error, setError] = useState<{ message: string } | undefined>(
+    undefined,
+  );
+
+  useEffect(() => {
+    if (!result && !error) executeExample();
+  }, [result, error]);
+
+  const executeExample = async (): Promise<void> => {
```

Comment:
Usually, async function definition goes into `useEffect` or even better outside, in a separate file. This is just to prevent the function to be redeclared in each render. Since this is just a demo moving the function inside the `useEffect` should be enough.

Reply:
Thanks! Updated https://github.com/regen-network/regen-js/pull/82/commits/e02cfd1cae434dac2b383df0908556117f5b3827